blocks/blockstore/blockstore_test.go

```diff
@@ -57,16 +57,21 @@ func TestRuntimeHashing(t *testing.T) {
 	bs := NewBlockstore(ds_sync.MutexWrap(ds.NewMapDatastore()))
 	bl := blocks.NewBlock([]byte("some data"))
 	blBad, err := blocks.NewBlockWithHash([]byte("some other data"), bl.Key().ToMultihash())
+	bl2 := blocks.NewBlock([]byte("some other data"))
```

Comment:
still, you want to check errors right as they are created, otherwise it just looks sloppy.